tests/testthat/test-create_codemeta.R

```diff
@@ -0,0 +1,36 @@
+test_that("create_codemeta works", {
+  tmp <- tempdir(check = TRUE)
+
+  cm <- create_codemetas(folder = testthat::test_path("packages"))
+  jsonlite::write_json(
+      cm,
+      file.path(tmp, "raw_cm.json"),
+      pretty = TRUE,
+      auto_unbox = TRUE
+    )
+
+  testthat::expect_snapshot_file(file.path(tmp, "raw_cm.json"))
+
+  dir.create(file.path(tmp, "packages"))
+  file.copy(
+    testthat::test_path("packages"),
+    tmp,
+    recursive = TRUE)
+
+  # case where the package source is no longer ok, use previous entry
```

Comment:
this tests whether we do not remove from the codemeta.json a repo whose man folder was deleted (which was why you opened the issue @sckott)

Reply:
perfect